src/HashBus.Application/Program.cs

```diff
@@ -8,10 +8,11 @@ class Program
         static void Main()
         {
             var nserviceBusConnectionString = ConfigurationManager.AppSettings["NServiceBusConnectionString"];
-
+            var endpointName = ConfigurationManager.AppSettings["EndpointName"];
+			

```

Comment:
This line has 3 tabs in it. If they can be removed, then I believe this PR is good to go (with a squash).